extensions/interactions/RatioExpressionInput/RatioExpressionInput.py

```diff
@@ -49,7 +49,7 @@ class RatioExpressionInput(base.BaseInteraction):
     }, {
         'name': 'numberOfTerms',
         'description': (
-            'The number of elements that the answer must have.'
+            'The minimum number of elements that the answer must have.'
```

Comment:
Actually this should be the *actual* number of terms, not the minimum.

Reply:
Done, though doesn't that mean the 'HasNumberOftermsEqualTo' rule is now redundant?